source/KadeEngineData.hx

```diff
@@ -5,7 +5,10 @@ class KadeEngineData
 {
     public static function initSave()
     {
-        if (FlxG.save.data.newInput == null)
+		if (FlxG.save.data.weekUnlocked == null)

```

Comment:
why is spacing inconsistent,,, file uses 4 spaces at the start then uses tabs,,,, if this was a problem before dont worry but if ur pr did it then pls fix

Reply:
will fix, didn't know that the spaces were intentional